http_parser.c

```diff
@@ -472,6 +484,107 @@ static struct {
 
 int http_message_needs_eof(const http_parser *parser);
 
+#if defined(_MSC_VER) // SSE2 is baseline
+#include <intrin.h>
+#define __builtin_ctz _tzcnt_u32
+#elif defined(__SSE2__) && defined(__GNUC__)
+#include <emmintrin.h>
+#endif
+
+#ifndef USE_INTRISICS_CRLF
+# if defined(_MSC_VER) || \
+    (defined(__SSE2__) && defined(__GNUC__))
+#   define USE_INTRISICS_CRLF 1
+# endif
```

Comment:
This is probably a rarity, but perhaps we should put an `else` here and set the define to 0.